projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -31,6 +31,12 @@ export default function SubscribePanels() {
 		} );
 	}, [] );
 
+	// Can be “private”, “password”, or “public”.
+	const postVisibility = useSelect( select => {
```

Comment:
Just a note here that the `postVisibilty` doesn't update to "password" when the password is selected. The update to "password" occurred when a character was typed in the input field